pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -23,13 +23,15 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-const parallelism = 16
+// Parallelism is used the store value from scheduler config
+var Parallelism = 16
 
 // chunkSizeFor returns a chunk size for the given number of items to use for
 // parallel work. The size aims to produce good CPU utilization.
```

Comment:
Also say:

returns max(1, min(sqrt(n), n/Parallelism))

Reply:
Do you want me to convert it to a named return or add a new function ?

Reply:
no, just add a comment to the function header.